diff --git a/Users/ryan/lib/geoportal/www/catalog/skins/tiles/footer.jsp b/footer.jsp
index 34604ea..cb5314f 100644
--- a/Users/ryan/lib/geoportal/www/catalog/skins/tiles/footer.jsp
+++ b/footer.jsp
@@ -16,4 +16,32 @@
 <%@taglib prefix="f" uri="http://java.sun.com/jsf/core"%>
 <%@taglib prefix="h" uri="http://java.sun.com/jsf/html"%>
 
-<h:outputText id="gptFooterStatement" escape="false" value="#{gptMsg['catalog.content.footer.statement']}"/>
\ No newline at end of file
+<div style='height: 120px; padding-bottom: 5px;'>
+	<h:outputText id="gptFooterStatement" escape="false" value="#{gptMsg['catalog.content.footer.statement']}"/>
+	<div style='width: 100%; padding-top: 5px;'>
+		<table width="100%" cellpadding="0" cellspacing="0">
+			<tr>
+				<td width="20%" align='left'>
+					<h:graphicImage value="/catalog/images/doe_banner_50h.png" 
+						style=''
+						alt="Produced under U.S. Department of Energy award DE-EE0001120 to Boise State University and U.S. Department of Energy award DE-EE0002850 to the Arizona Geological Survey acting on behalf of the Association of American State Geologists. "
+						title="Produced under U.S. Department of Energy award DE-EE0001120 to Boise State University and U.S. Department of Energy award DE-EE0002850 to the Arizona Geological Survey acting on behalf of the Association of American State Geologists. "/>
+				</td>
+				<td width="80%" align='left'>
+					Produced under U.S. Department of Energy award DE-EE0001120 to Boise State University and U.S. Department of Energy award DE-EE0002850 to the Arizona Geological Survey acting on behalf of the Association of American State Geologists.  
+				</td>
+			</tr>
+			<tr>
+				<td width="20%" align='left'>
+					<h:graphicImage value="/catalog/images/nsf_banner_50h.png" 
+						style='' 
+						alt="Development of this catalog supported by the National Science Foundation under EAR-0753154 to the Arizona Geological Survey acting on behalf of the Association of American State Geologists"
+						title="Development of this catalog supported by the National Science Foundation under EAR-0753154 to the Arizona Geological Survey acting on behalf of the Association of American State Geologists"/>
+				</td>
+				<td width="80%" align='left'>
+					Development of this catalog supported by the National Science Foundation under EAR-0753154 to the Arizona Geological Survey acting on behalf of the Association of American State Geologists
+				</td>
+			</tr>
+		</table>
+	</div>
+</div>
\ No newline at end of file
